app/javascript/hyacinth_ui_v1/components/digital_objects/new/AssetNew.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form } from 'react-bootstrap';
+import { useMutation } from '@apollo/react-hooks';
+import uploadFile from '../../../graphql/file_uploads';
+import { createAssetMutation } from '../../../graphql/assets';
+
+function debugObject(obj) {
```

Comment:
It looks like this isn't being used anywhere, but I could be wrong.

Reply:
it's really just there for debugging!

Reply:
Makes sense!  I just mean probably good to remove before we merge this PR.

Reply:
Removed.